### PR TITLE
RESOURCE-129 Support service bus namespaces

### LINKED ISSUE #552: Service Bus Namespaces

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description

Support Service Bus Namespace(s)

### Issues Resolved
#552 

### Check List

- [X] New functionality includes tests
- [X] All tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: libraries/azure_service_bus_namespace.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require 'azure_generic_resource'
+
+class AzureServiceBusNamespace < AzureGenericResource
+  name 'azure_service_bus_namespace'
+  desc 'Retrieves and verifies the settings of an Azure Service Bus Namespace.'
+  example <<-EXAMPLE
+    describe azure_service_bus_namespace(resource_group: 'migrated_vms', name: 'inspec_ns') do
```
Comment (soumyo13) [inline libraries/azure_service_bus_namespace.rb:7]:
```suggestion
    describe azure_service_bus_namespace(resource_group: 'RESOURCE_GROUP',  name: 'SERVICE_BUS_NAMESPACE') do
```

File: libraries/azure_service_bus_namespaces.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+require 'azure_generic_resources'
+
+class AzureServiceBusNamespaces < AzureGenericResources
+  name 'azure_service_bus_namespaces'
+  desc 'Verifies settings for a collection of Azure Service Bus Namespaces in a Resource Group'
+  example <<-EXAMPLE
+    describe azure_service_bus_namespaces(resource_group: 'migrated_vms') do
```
Comment (soumyo13) [inline libraries/azure_service_bus_namespaces.rb:7]:
```suggestion
    describe azure_service_bus_namespaces(resource_group: 'RESOURCE_GROUP') do
```

**Review by soumyo13 — COMMENTED** [review]
LGTM

**Review by soumyo13 — APPROVED** [review]

File: libraries/azure_service_bus_namespace.rb (hunk 1 above)
Comment (sathish-progress) [reply libraries/azure_service_bus_namespace.rb:7]: as discussed offline we don't follow this construct in ruby files

File: libraries/azure_service_bus_namespaces.rb (hunk 2 above)
Comment (sathish-progress) [reply libraries/azure_service_bus_namespaces.rb:7]: not following this construct in rb files

**Review by IanMadd — APPROVED** [review]

**Pull request merged**